test/Interpreter/protocol_composition_with_markers.swift

```diff
@@ -27,3 +27,27 @@ do {
   print(v1 == v2)
   // CHECK: true
 }
+
+@_marker
+protocol Marker {
+}
+
+do {
+  print(G<any (C & Sendable)>.self)
+  // CHECK: G<C>
+
+  class D<T> {
+  }
+
+  print((D<Int> & Sendable).self)
```

Comment:
You still need to test this from generic context, ie

```
func f<T>(_: T.Type) {
  print(D<T> & Sendable.self)
}
```

Reply:
Ah sure!

Reply:
✅